src/osdatahub/NGDAPI/datasets.py

```diff
@@ -0,0 +1,32 @@
+from collections import namedtuple
+
+Dataset = namedtuple("Dataset", "name short_code")
+
+# TODO: add rest of collections
+# TODO: change datasets to collections
+DATASETS = {
+    "building_line": Dataset("Building Line", "bld-fts-buildingline"),
+    "building_part": Dataset("Building Part", "bld-fts-buildingpart"),
+    "named_area": Dataset("Named Area", "gnm-fts-namedarea"),
+    "named_point": Dataset("Named Point", "gnm-fts-namedpoint"),
+
+}
+
+SHORT_CODES = {
```

Comment:
Is this set up so that users can enter short codes or names? If so, would users need to give each code separately or would you do a string split? Wondering if it would be easier not to have the nested structure!

Reply:
With short codes, I was just experimenting. Ben and I were thinking that the NGD team might add support for just the theme short codes in the future, in which case a nested structure would be good for reducing code replication and allow people to add a wider variety of short codes. However, that isn't implemented yet so we weren't sure how to go about it just yet